cleanslate/cleanslate/urls.py

```diff
@@ -19,12 +19,13 @@
 from django.views.generic import RedirectView
 from django.conf import settings
 from django.conf.urls.static import static
+from django.contrib.auth import views as auth_views
 from doma.views import home, profile, calendar, reminders, finance, edit_chore_deadline, create_chore, delete_chore, EditUserProfileView
 
 urlpatterns = [
-    url(r'^$', home, name='doma/login/'),
     url(r'^admin/', admin.site.urls),
     url(r'^doma/$', home, name='doma'),
+    url(r'^$', auth_views.login, name='doma-login'),
```

Comment:
These lines seem to be in two different pull requests, definitely avoid doing this. Keep new code to just one PR and isolate your projects

Reply:
Could you elaborate a bit? 
You're saying this pr includes the code from the previous, correct? If so, what is the best method to "isolate". 
Learning git methodologies like this would be very useful for me, as problems like this seem to continuously arise from my lack of knowledge in this area.

Reply:
This is why I had offered to teach git earlier in the semester multiple times but unfortunately we never met to do so. I believe you understood this when I explained in slack, regardless, this line shouldn't be necessary in either PR and I have removed it in my most recent PR #73